internal/inventory/ASSETS.md

```diff
@@ -0,0 +1,177 @@
+| Infrastructure | Integration | API | API Gateway Stage | No ❌ |
+| Infrastructure | Integration | Event Source | Lambda Event Source Mapping | No ❌ |
+| Infrastructure | Integration | Message Queue | SQS Queue | No ❌ |
+| Infrastructure | Management | Cloud Account | Cloud Service Provider Account | No ❌ |
+| Infrastructure | Management | Configuration | Config Configuration Recorder | No ❌ |
+| Infrastructure | Management | Configuration | Config Delivery Channel | No ❌ |
+| Infrastructure | Management | Inventory | Systems Manager Instance | No ❌ |
+| Infrastructure | Management | Patch | Systems Manager Instance Patch | No ❌ |
+| Infrastructure | Network | DNS | DNS Record | No ❌ |
+| Infrastructure | Network | DNS | Nameserver | No ❌ |
+| Infrastructure | Network | DNS | Route53 DNS Record | No ❌ |
+| Infrastructure | Network | DNS | Route53 DNS Zone | No ❌ |
+| Infrastructure | Network | Endpoint | Network Endpoint | No ❌ |
+| Infrastructure | Network | Firewall Rule | Inbound IP Permission | No ❌ |
+| Infrastructure | Network | Firewall Rule | IP Rule | No ❌ |
+| Infrastructure | Network | Firewall | EC2 Security Group | No ❌ |
+| Infrastructure | Network | Gateway | Internet Gateway | No ❌ |
+| Infrastructure | Network | Interface | EC2 Network Interface | Yes ✅ |
+| Infrastructure | Network | IP Address Range | VPC CIDR Block | No ❌ |
+| Infrastructure | Network | IP Address Range | VPC IPv4 CIDR Block | No ❌ |
+| Infrastructure | Network | IP Address Range | VPC IPv6 CIDR Block | No ❌ |
+| Infrastructure | Network | IP Address | EC2 Private IP | No ❌ |
+| Infrastructure | Network | IP Address | Elastic IP | No ❌ |
+| Infrastructure | Network | IP Address | IP Address | No ❌ |
+| Infrastructure | Network | Load Balancer | ELBv2 Listener | No ❌ |
+| Infrastructure | Network | Load Balancer | Elastic Load Balancer | Yes ✅ |
+| Infrastructure | Network | Load Balancer | Elastic Load Balancer v2 | Yes ✅ |
+| Infrastructure | Network | Load Balancer | ELB Listener | No ❌ |
+| Infrastructure | Network | Peering | VPC Peering Connection | No ❌ |
+| Infrastructure | Network | Subnet | DB Subnet Group | No ❌ |
+| Infrastructure | Network | Subnet | EC2 Subnet | Yes ✅ |
+| Infrastructure | Network | Virtual Network | Transit Gateway | Yes ✅ |
+| Infrastructure | Network | Virtual Network | Transit Gateway Attachment | Yes ✅ |
+| Infrastructure | Network | Virtual Network | VPC | Yes ✅ |
+| Infrastructure | Security | Encryption | KMS Key | No ❌ |
+| Infrastructure | Security | Encryption | KMS Key Alias | No ❌ |
+| Infrastructure | Security | Secrets Management | Secrets Manager Secret | No ❌ |
+| Infrastructure | Security | Security Management | Security Hub | No ❌ |
+| Infrastructure | Storage | Disk | EBS Volume | No ❌ |
+| Infrastructure | Storage | Object Storage | S3 Bucket | No ❌ |
+| Infrastructure | Storage | Snapshot | EBS Snapshot | No ❌ |
+| Infrastructure | Storage | Snapshot | RDS Snapshot | No ❌ |
+
+Progress: 12% (10/83)
```

Comment:
Can the progress be on top? Can we break it down per resource type?

Reply:
Added, broken down by category.